Jenkinsfile

```diff
@@ -34,7 +34,8 @@ def generateStages(String os, def mvnVersion, def javaVersion) {
     def prefix = isWindows ? "win" : "linux"
     def nodeLabel = isWindows ? "Windows" : "ubuntu"
     
-    String goals = (!isWindows && shouldDeploy()) ? 'clean deploy' : 'clean install'
+    boolean shouldDeployInThisBranch = (!isWindows && shouldDeploy())
+    String goals = shouldDeployInThisBranch ? 'clean deploy' : 'clean install'
```

Comment:
@kwin - I am not sure we need this. We deploy the p2 update site via SSH upload but I don't think the shared Maven artifacts are used anywhere.